step-templates/mysql-backup-database.json

```diff
@@ -0,0 +1,105 @@
+      "Octopus.Action.EnabledFeatures": ""
+    },
+    "Parameters": [
+      {
+        "Id": "7030ad54-4e10-4854-998a-ca91e3b490a3",
+        "Name": "createMySQLServerName",
+        "Label": "Server",
+        "HelpText": "Name or IP of the MySQL server",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "07dbf251-a982-4cc5-b6c1-dc0af50c519c",
+        "Name": "createusername",
+        "Label": "Username",
+        "HelpText": "\tUsername with rights to dump database",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "ac42a150-b7e7-4414-9dec-f18e105eaedf",
+        "Name": "createUserPassword",
+        "Label": "Password",
+        "HelpText": "Password for the user account",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "Sensitive"
+        }
+      },
+      {
+        "Id": "62ffb949-c5db-4151-bb29-47a37586bda5",
+        "Name": "createDatabaseName",
+        "Label": "Database Name",
+        "HelpText": "Name of the database to backup",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "c54832b1-571f-4475-9159-6a7342a67982",
+        "Name": "createPort",
+        "Label": "Port",
+        "HelpText": "Port number for the MySQL server",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "SingleLineText"
+        }
+      },
+      {
+        "Id": "a33dc26c-15b3-4b5e-a62f-7a5e97576bcc",
+        "Name": "createUseSSL",
+        "Label": "Use SSL",
+        "HelpText": "Check this box to force the use of SSL.",
+        "DefaultValue": "",
+        "DisplaySettings": {
+          "Octopus.ControlType": "Checkbox"
+        }
+      },
+      {
+        "Id": "fc2c1a2f-43ba-4878-8e74-b88dad96c5d2",
+        "Name": "createmysqlpath",
+        "Label": "MySQL Path",
+        "HelpText": "Path to binaries e.g. C:\\Program Files\\MySQL\\MySQL Server 5.6\\bin",
+        "DefaultValue": "C:\\Program Files\\MySQL\\MySQL Server 5.6\\bin",
```

Comment:
Perhaps this needs an explicit call out in the description markdown - that the mysql binaries are required and if not present, it will error?